test/e2e/specs/test.spec.js

```diff
@@ -102,20 +102,27 @@ describe('Wallet App Test Cases', { execTimeout: DEFAULT_TIMEOUT }, () => {
       });
     });
 
-    it('should place a bid by discount from the CLI successfully and verify IST balance', () => {
-      cy.addNewTokensFound();
-      cy.getTokenAmount('IST').then((initialTokenValue) => {
-        cy.placeBidByDiscount({
-          fromAddress: accountAddresses.user2,
-          giveAmount: '2IST',
-          discount: 5,
-        }).then(() => {
-          cy.getTokenAmount('IST').then((tokenValue) => {
-            expect(tokenValue).to.lessThan(initialTokenValue);
+    it(
+      'should place a bid by discount from the CLI successfully and verify IST balance',
+      {
+        defaultCommandTimeout: DEFAULT_TIMEOUT,
```

Comment:
Is there a way to set these defaults globally?

Reply:
We can set these globally in `synpress.config.cjs`. Do you think it's a good idea to set them globally? Currently, we are only using these defaults in tests where they are specifically required

Reply:
well the values are called `DEFAULT_TIMEOUT` and `DEFAULT_TASK_TIMEOUT`. If they're not really the default values, then please change the name. E.g. `SLOW_TIMEOUT`. Then update the config with the actual desired defaults.